### ISSUE TITLE
why did the results always predict the same output

### ISSUE THREAD

**Issue by febryardiansyah** [SashiDo/teachablemachine-node#3]
Hi !!, I just following your sample code and using your sample model.
but, when i change the image url. the output always same as before.
Thanks.

the outputs:

```json
Predictions: [
  { className: 'Dog', probability: 0.5297428965568542 },
  { className: 'Not_Dog', probability: 0.47025713324546814 }
]
current imageUrl : https://media-blog.sashido.io/content/images/2020/09/SashiDo_Dog.jpg
```

```json
Predictions: [
  { className: 'Dog', probability: 0.5297428965568542 },
  { className: 'Not_Dog', probability: 0.47025713324546814 }
]
current imageUrl : https://icatcare.org/app/uploads/2018/07/Thinking-of-getting-a-cat.png
```

**Comment by veselinastaneva** [comment]
Hi Febry,

Thank you very much for reporting this!

Our team is already on it and I'll keep you posted once we get it fixed. 

In the meantime, I would love to learn what kind of project do you plan to use this for. Is there anything else we can add to make it more helpful?

Any other feedback will be much appreciated! :)

**Comment by pivanov** [comment]
Hey @febryardiansyah,

Thank you very much for reporting this!
The the issue was fixed and [v1.1.1](https://www.npmjs.com/package/@sashido/teachablemachine-node) is ready 

Can you check it and if anything came up please let us know.
